htdocs/install/mysql/migration/3.7.0-3.8.0.sql

```diff
@@ -797,3 +797,14 @@ ALTER TABLE llx_societe_remise_except MODIFY COLUMN description text NOT NULL;
 update llx_opensurvey_sondage set format = 'D' where format = 'D+';
 update llx_opensurvey_sondage set format = 'A' where format = 'A+';
 
+--Deal with holidays_user that do not have rowid
```

Comment:
I reverted that. No database structure change is allowed into a minor version.
Why this change ? A rowid is not required here, the key is couple fk_user/fk_type and code should rely on this.